src/komponenter/kalkulator/kalkulator.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Kalkulator from './kalkulator';
+
+test('Tester at kalkulator renderes med basic innhold', () => {
+    const { getByText } = render(<Kalkulator />);
+
+    const permitteringsElement = getByText(
+        /Legg inn permitteringsperiode for arbeidstaker/i
```

Comment:
Det er fort gjort at denne teksten endrer seg, hva med å hente et react-element i stedet?

Reply:
Ja, fast egentlig så er det enligt "guideline" på det rammeverket som react app bruker at man skal prioritere at teste på text med argumentet: "testing the application components in the way the user would use it.". De har ikke lagt noe spesifik metode for å hente elementer av den grunnen.

Reply:
Oi! Overraskende 😅

Reply:
Ja, men se poengen i denne type komponenttest. (litt ihvertfall :p)